cmd/operator/README.md

```diff
@@ -26,11 +26,9 @@ The default operator deployment depends on NFD and cert-manager. Those component
 Install NFD (if it's not already installed) and node labelling rules (requires NFD v0.10+):
```

Comment:
As this now depends on a newer NFD, it would be good to mention the version here?

Reply:
yes makes sense

Reply:
fixed